co-log-core/src/Colog/Core/Action.hs

```diff
@@ -112,6 +114,23 @@ instance Applicative m => Monoid (LogAction m a) where
     mconcat = foldActions
     {-# INLINE mconcat #-}
 
+
+{- | Operator version of 'unLogAction'
+
+Note that because of the types, something like:
+> action <& msg1 <& msg2
+doesn't make sense. Instead you want:
+> action <& msg1 >> action <& msg2
+-}
+infix 5 <&
+(<&) :: LogAction m msg -> msg -> m ()
+(<&) = unLogAction
+
+-- | A flipped version of <&
```

Comment:
To have it showed as the link to the `<&` function in the haddock documentation you should put it into quotes: `'<&'`

Reply:
Right